tests/attr/layer/test_layer_gradient_shap.py

```diff
@@ -112,33 +117,33 @@ def test_basic_multi_input(self):
 
     def _assert_attributions(
         self,
-        model,
-        layer,
-        inputs,
-        baselines,
-        target,
-        expected,
-        expected_delta=None,
-        n_samples=5,
-        attribute_to_layer_input=False,
-        add_args=None,
-    ):
+        model: Module,
+        layer: Module,
+        inputs: TensorOrTupleOfTensors,
+        baselines: Union[TensorOrTupleOfTensors, Callable],
+        target: Union[int, Tuple[int, ...], Tensor, List[Tuple[int, ...]]],
+        expected: Union[Tensor, Tuple[Tensor, ...], List, Tuple[List, ...]],
```

Comment:
nit: Could make the List hint more specific? (List[float] or List[List[float]]?)

Reply:
It makes it a little more lengthy but added it